### PR TITLE
Wal 247

### PR DESCRIPTION
**Pull request by mseaward**
Changed
remove GPOS Vested Balance types from "Vesting Balances (Witness Pay)" section (WAL-246)
change text of "Vesting Balances (Witness Pay)" to "Pending Balances" (WAL-255)
Added
GPOS Panel (WAL-247, WAL-245)
help modal dummy extended FAQ content (WAL-253)
display project code version in top middle of application (WAL-256)

### CODE REVIEW

File: src/components/GPOSPanel/GPOSPanel.jsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+          <Translate content='gpos.side.info.percent'/>
+          <div className='gpos-panel__stats--right'>
+            {gposPerformance}%
+          </div>
+        </div>
+
+        <div className='gpos-panel__stats-potential'>
+          <Translate content='gpos.side.info.potential'/>
+          <div className='gpos-panel__stats--right'>
+            {gposAward && asset
```
Comment (aametzler) [inline src/components/GPOSPanel/GPOSPanel.jsx]: potentially rename from gposAward to gposReward

File: src/components/Help/HelpModal.jsx [hunk 2]
```diff
@@ -1,14 +1,26 @@
-
 import React from 'react';
-import ReactDOM from 'react-dom';
+// import ReactDOM from 'react-dom';
```
Comment (aametzler) [inline src/components/Help/HelpModal.jsx]: remove instead of comment

File: src/components/Help/HelpModal.jsx [hunk 3]
```diff
@@ -72,53 +83,61 @@ class HelpModal extends React.Component {
                       component='a'
                       className='help__link'
                       content='help.anchors.my_funds'
-                      onClick={ this.scrollToHeaderByRefName.bind(this, 'fundsAnchor') }/>
+                      // onClick={ () => this.scrollToHeaderByRefName(this, 'fundsAnchor') }/>
```
Comment (aametzler) [inline src/components/Help/HelpModal.jsx]: remove unused

File: src/components/Utility/BindToChainState.jsx [hunk 4]
```diff
@@ -62,7 +63,7 @@ function BindToChainState(options) {
     return class Wrapper extends React.Component {
       constructor(props) {
         super(props);
-        let prop_types_array = pairs(Component.propTypes); // TODO: deprecate
+        let prop_types_array = toPairs(Component.propTypes); // TODO: deprecate
```
Comment (aametzler) [inline src/components/Utility/BindToChainState.jsx]: remove TODO comment

File: src/constants/ActionTypes.js [hunk 5]
```diff
@@ -251,62 +249,67 @@ export default {
...
      */
-    TOGGLE_HELP_POPUP : 'TOGGLE_HELP_POPUP',
+  TOGGLE_HELP_POPUP : 'TOGGLE_HELP_POPUP',
+
+  /**
+     * GPOS Wizard Modal
+     *
+     * const TOGGLE_GPOS_WIZARD - Show|hide GPOS Wizard modal.
+     */
+  TOGGLE_GPOS_WIZARD : "TOGGLE_GPOS_WIZARD",
```
Comment (aametzler) [inline src/constants/ActionTypes.js]: use single quotes

File: src/components/Utility/BindToChainState.jsx (hunk 4 above)
Comment (mseaward) [reply src/components/Utility/BindToChainState.jsx]: updated comment with more context

**Pull request merged**